src/client/pages/Registration.tsx

```diff
@@ -42,40 +103,46 @@ export const Registration = ({ returnUrl = '', email = '' }: Props) => {
           },
         ]}
       />
-      <Main>
+
+      <Main
+        errorOverride={
+          recaptchaCheckSuccessful ? undefined : CaptchaErrors.GENERIC
+        }
+      >
         <PageBox>
           <PageBody>
             <form
-              css={form}
+              css={formSpacing}
               method="post"
-              action={`${Routes.REGISTRATION}${returnUrlQuery}`}
+              action={`${Routes.REGISTRATION}?${registrationUrlQueryParamString}`}
+              ref={registerFormRef}
+              onSubmit={handleSubmit}
             >
               <CsrfFormField />
+              <RecaptchaElement
+                ref={recaptchaElementRef}
+                id="register-recaptcha"
+              />
               <TextInput
                 css={textInput}
                 label="Email"
                 name="email"
                 type="email"
                 defaultValue={email}
               />
-              <TextInput
-                css={textInput}
-                label="Password"
-                name="password"
-                type="password"
-              />
+              <div css={termsSpacing}>
+                <Terms />
+              </div>
               <Button css={button} type="submit" data-cy="register-button">
                 Register
               </Button>
             </form>
             <Divider
-              size="full"
-              spaceAbove="loose"
+              size="fit"
+              spaceAbove="none"
```

Comment:
`none` gives us the ability to responsively control the margin using `formSpacing` above.https://github.com/guardian/gateway/pull/955#diff-6667223d0620884563d10ac89c87f3ca30726a21bc2d4dc5719ba3d696e455d0R38